### ISSUE TITLE
Measure - Multiple `<video>` `<source>` tags all reported as downloaded

### ISSUE THREAD

**Issue by huntharo** [GoogleChrome/lighthouse#14291]

**Comment by adamraine** [comment]
Yeah this is happening on web.dev and PSI but never DevTools or CLI.

I tried launching Chrome with `--headless` and `--headless=chrome` on the CLI but it still wouldn't repro.

**Comment by ErwinAI** [comment]
Having the same problem here!

<img width="708" alt="image" src="https://github.com/GoogleChrome/lighthouse/assets/2168688/407c2b46-c09d-432f-a758-41729accd4c3">
